stylus/include/Home.styl

```diff
@@ -15,31 +17,25 @@
 		margin-left: 25rem
 		margin-right: 25rem
 		padding-bottom: 3rem
-	>div:nth-child(5), div:nth-child(6)
+	>.calendar-images
```

Comment:
No need to use a child selector here. I would generally only use it when you need to distinguish between the styles of an element (say, a div) that is a direct child of its parent vs. that same element (another div) in a different position in the html. This will usually not be necessary for classes.

Reply:
Also, to expand a bit more on classes:  ideally, they should be reusable, like if there ends up being another view on the site that uses rows of images like this, you could just have an `.image-rows` class and use it in both places. But even if that doesn't happen, I'm fine with having the occasional class name that isn't reused so long as it helps divide up the css in a logical, readable fashion rather than using a bunch of vague nth-child type stuff. No need to change anything here. Just wanted to expound a bit.